lib/elixir/lib/code.ex

```diff
@@ -1192,6 +1192,24 @@ defmodule Code do
     :elixir.string_to_quoted!(to_charlist(string), line, column, file, opts)
   end
 
+  @doc """
+  Converts a quoted expression to an algebra document.
+
+  Supports both regular AST and the extended AST documented in
+  `string_to_quoted_with_comments/2`.
```

Comment:
I think this comment is now a little outdated. Instead, let's document you get better results if the quoted expression is built with token_metadata and a literal block encoder:

```elixir
    [
      literal_encoder: &{:ok, {:__block__, &2, [&1]}},
      token_metadata: true
    ]
```

Reply:
Also, if we want to support unescape, then we need to list it too. See this comment for more info: https://github.com/elixir-lang/elixir/pull/10988/files#r636739871